src/meta-srv/src/bootstrap.rs

```diff
@@ -225,8 +227,14 @@ pub async fn metasrv_builder(
         (None, BackendImpl::PostgresStore) => {
             let pg_client = create_postgres_client(opts).await?;
             let kv_backend = PgStore::with_pg_client(pg_client).await.unwrap();
-            // TODO(jeremy, weny): implement election for postgres
-            (kv_backend, None)
+            let election_client = create_postgres_client(opts).await?;
```

Comment:
We have to create another client since `tokio_postgres::Client` is stateful and cannot be cloned. Would it be possible that one of the connections crashed and cause unpredictable fault?

Reply:
I think it's ok. If this election client is broken, the lock is auto released thanks to this advisory lock mechanism. And if the kv client is broken, it will try to reconnect on next request.